Backend/common.js

```diff
@@ -85,15 +87,23 @@ exports.defaultError = defaultError;
 // <Global Constants> ------------------------------------------
 // all user types
 const userTypes = Object.freeze({
-    MODE_SELECTOR: 0,
-    COLLABORATOR_ADMIN: 1,
-    COLLABORATOR: 2,
-    PROFESSOR: 3,
-    TA: 4,
-    STUDENT: 5
+    MODE_SELECTOR: { value: 0, text: 'Mode Selector' },

```

Comment:
we should not be adding text because then it will not follow the translations (unless it's not being passed down to the UI)

Reply:
same for the other objects here

Reply:
As we talked offline, I removed those from the pugs and used translation instead. But I will keep this for now. Its not linked to the UI anymore